ansible/roles/codeready-containers-hacks/tasks/delete-code-ready-containers.yml

```diff
@@ -0,0 +1,15 @@
+- name: Delete CodeReady Deployment
+  command: "crc delete -f"
+  ignore_errors: yes
+
+- name: get the username running the deploy
+  become: false
+  command: whoami
+  register: username
+
+- debug: var=username.stdout
+
+- name: Clean crc directory
+  file:
+    state: absent
+    path: "/home/{{ username.stdout }}/.crc"
```

Comment:
The use of `whoami` should not be needed here. Should work fine with just using the home directory `~` marker.

```
- name: Clean crc directory
  file:
    state: absent
    path: "~/.crc"
```